src/test/java/io/pivotal/literx/Part03StepVerifierTest.java

```diff
@@ -23,6 +23,11 @@
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
+// to use inside expectSkylerJesseComplete
```

Comment:
this is not shown in the playground, so not terribly relevant. Plus I think the `*` imports clash (both have `assertThat` methods)

Reply:
Actually I just checked `assertThat()` method with both imports JUnit and AssertJ 
in the playground, and it works without conflicts 
( I think because each library method has its own signature ), 
even without fully qualified name, 
except one thing that 
this `import static org.hamcrest.CoreMatchers.*;` must be added as well .

Reply:
ah I see :) already merged the PR but maybe it would benefit from preparing one static * import - let's say AssertJ - and mentioning one can use AssertJ (giving example of `assertThat(something).isEqualTo(something)` syntax) in the playground's lesson file. I'll do a commit to that effect.

Reply:
done

Reply:
@simonbasle thanks for the patience, review and feedback with this PR . 
I do not contribute very often . 
It turns out that this was my 1-st meaningful pull request .

Reply:
of course! glad your first meaningful PR was on one of our projects ;)